test/facebook-video.js

```diff
@@ -28,7 +28,7 @@ describe('Vimeo Video', () => {
     assert.strictEqual(t.canonicalUrl, 'https://www.facebook.com/heiseonline/videos/10156049485672318/')
   })
 
-  it('should provide the poster image', async () => {
+  it.skip('should provide the poster image', async () => {
```

Comment:
This test can probably be removed.

Reply:
removed in https://github.com/heiseonline/embetty-base/pull/20/commits/0897af1bac64c22341ce308a33edabde49acb881